radio/src/gui/9x/menu_model_setup.cpp

```diff
@@ -679,7 +679,7 @@ void menuModelSetup(uint8_t event)
               break;
             case 1:
               CHECK_INCDEC_MODELVAR(event, g_model.ppmNCH, -2, 4);
```

Comment:
I ran into a issue trying to fix this problem. the 9x files are used by the avr targets as well as by the Sky9x target. Adding ppm struct the 9x modeldata will not work and ifdefs in the menu_model_setup.cpp are probably also not the best idea. Any idea how to best proceed?

Maybe using a macro PPM(delay) that expands to the one or other variants depending on the platform?